specs/2019-04/Windows-Plan.md

```diff
@@ -0,0 +1,96 @@
+# App Center Window’s Proposal 
+
+This plan details the proposed improvements for App Center’s Windows offering along with a comparison of App Center and HockeyApp. This is work in progress and will be iterated upon based on any internal updates and feedback from the community. We expect to start work on the planned items in the second quarter of 2019. 
+
+
+## App Center’s current Windows Offering 
+### Diagnostics
+
+- Crash reporting for UWP (.NET native) apps distributed through the Windows store 
+> Note: there are some known limitations and missing features for UWP apps. Find more details in our [Windows documentation](https://docs.microsoft.com/en-us/appcenter/diagnostics/windows-support). 
+
+### Distribution
+
+- Distributing UWP and Unity UWP apps 
+- Unofficial support for distributing WPF (.NET Framework) applications 
+
+### Analytics 
+
+- Analytics SDK for UWP apps and all of it’s functionalities listed [here](https://docs.microsoft.com/en-us/appcenter/analytics/).  
+
+### Build
+- Build support for UWP apps
+
+## Planned Improvements 
+
+### Diagnostics 
+
+- Crash reporting capabilities for store and sideloaded UWP and WPF apps aligned with other platforms supported in App Center. Find more details on App Center Diagnostics features in our documentation.  
+
+> Note: this does not include handled exceptions and custom properties.  
+
+### Distribution 
+
+- Official support for WPF applications 
+ 
+> Note: this does not include in app updates through App Center.  
+### Analytics
+- Analytics SDK for WPF apps and all of it’s functionalities as listed [here](https://docs.microsoft.com/en-us/appcenter/analytics/).  
+
+## HockeyApp Comparison (Windows Specific) 
+### Diagnostics 
+Note: “full support” means supporting all App Center crashes features as indicated [here](https://docs.microsoft.com/en-us/appcenter/diagnostics/features), not including handled exceptions and custom key value pairs.  
+
+|    | HockeyApp | App Center Today | Proposal | 
+| ---| --------- |:----------------:| ------- |
+| UWP| Yes       | Partial support | Full support for crashes |
+| WPF| Yes       | None | Full support for crashes |
+| WinRT / Silverlight | Yes       | None | None |
+| WinForms | (Open Source)    | None | Stretch goal |
+| Win32 | None | None | None |
+
+
+### Distribution  
+
+Note: HockeyApp supports over the air app updates for WPF, WinRT, and Silverlight applications. App Center will consider supporting this scenario for distributing Windows apps using the .AppInstaller file (#52) 
+
+|    | HockeyApp | App Center Today | Proposal | 
+| ---| --------- |:----------------:| ------- |
+| UWP| Yes       | Yes | None (already supported) |
+| WPF| Yes       | None | Official support for .NET framework |
+| WinRT / Silverlight | Yes       | None | None |
```

Comment:
I believe @winnieli1208 is including this here because we support this in HockeyApp and have customers that still actively use that platform. It's important we set clear expectations with them so they can take necessary steps before the shutdown of HockeyApp at the end of this year.

Reply:
I see. Then I'd suggest to keep one table to explain HockeyApp compatibility/evolution, and switch to a new table where the new paradigms makes sense;
- Distribution. The app framework is not relevant. Just packaging with MSIX
- Analytics. I hope that we could get a common API with .NET Standardar that can be reused across different runtimes. (here the packaging is not relevant)
- Crashes. This feature is runtime/device specific and probably requires a mix of packaging + native capabilities.

..but I dont see any value describing the distribution options for WinRT/Silverlight ...